identity-service/src/notifications/formatNotificationMetadata.js

```diff
@@ -165,6 +165,14 @@ function formatReaction (notification, metadata) {
     amount: formatWei(new BN(notification.metadata.reacted_to_entity.amount))
   }
 }
+function formatReactionEmail (notification, extras) {
```

Comment:
Just so we remember late, maybe a comment that this operates on notifs from the db instead of dn (distinct from the other fn above)